src/utils/render-fragments.js

```diff
@@ -31,59 +166,104 @@ export function getAccessibilityRequirements(accessibility_requirements) {
 			<span role="heading" aria-level="1" className="heading">
 				Accessibility Requirements
 			</span>
-			{requirements.map(sc => {
-				const scData = scUrls[sc]
-				return (
-					<a className="sc-item" key={sc} href={scData.url}>
-						{scData.num} {scData.handle}
-					</a>
-				)
-			})}
+			<Accordion allowMultipleExpanded="true" allowZeroExpanded="true">
+				{requirements.map(sc => {
+					const scData = scUrls[sc]
+					return (
+						<AccordionItem key={sc}>
+							<AccordionItemHeading>
+								<AccordionItemButton>
+									<a className="sc-item" key={sc} href={scData.url}>
+										{scData.num} {scData.handle}
+									</a>
+								</AccordionItemButton>
+							</AccordionItemHeading>
+							<AccordionItemPanel>
+								<ul>
+									<li>
+										<strong>Required for conformance</strong>{' '}
```

Comment:
Note to self:
- Use format from here - https://www.w3.org/TR/act-rules-format/#outcome-mapping
eg: add sc level